linux_os/guide/system/software/integrity/fips/enable_fips_mode/ansible/shared.yml

```diff
@@ -4,5 +4,12 @@
 # complexity = medium
 # disruption = medium
 
-- name: enable fips mode
+- name: Check to see the current status of FIPS mode
+  command: /usr/bin/fips-mode-setup --check
```

Comment:
Not sure I quite fully agree with this approach. I think I'd prefer the simpler, cross-platform `/proc/sys/crypto/fips_enabled` check (if not-present or has value zero, FIPS is not enabled). My 2c.

Reply:
I don't have a strong opinion here, besides checking for `/proc/sys/crypto/fips_enabled` `fips-mode-setup --check` also checks for the selected crypto-policy.
The OVAL check ensures that the selected crypto-policy is FIPS, or FIPS with specific modules.
I'm slightly leaning towards `fips-mode-setup --check` as that is what is in the OCIL clause.

_Rant:_
When the system wide crypto-policy is changed this is not reflected in `/proc/sys/crypto/fips_enabled`, but  `fips-mode-setup --check` command shows the following:
```console
[root@localhost ~]# fips-mode-setup --check
FIPS mode is enabled.
The current crypto policy (DEFAULT) is not a FIPS policy.
```

But this may not matter much as the Ansible and Bash remediation don't set the crypto-policy to `var_system_crypto_policy`, :laughing:. The `fips-mode-setup --enable` command always selects FIPS policy.

Reply:
I agree with using `fips-mode-setup --check` the whole rule OCIL, description, existing fixes, all use `fips-mode-setup`.